stwcs/wcsutil/altwcs.py

```diff
@@ -97,21 +104,26 @@ def archiveWCS(fname, ext, wcskey=" ", wcsname=" ", reusekey=False):
                     wkey = next_wcskey(f[wcsext].header)
                 elif wkey is None:
                     closefobj(fname, f)
-                    raise KeyError("Could not get a valid wcskey from wcsname %s" % wcsname)
+                    raise KeyError(f"Could not get a valid wcskey from wcsname '{wcsname:s}'")
```

Comment:
Is the `:s` in `{wcsname:s}` a typo?

Reply:
It has the same effect as `%s` - it expects `wcsname` to be a string.